docs/assets/js/tabs.js

```diff
@@ -0,0 +1,51 @@
+import { Tabs } from 'govuk-frontend';
+
+/**
+ * @param {DOMNode} container - Parent element containing Gov UK tab HTML
+ */
+export function init( container ) {
+  const tabs = container.querySelectorAll( '[data-module="tabs"]' );
+
+  if ( tabs && tabs.length > 0 ) {
+    for ( let i = 0; i < tabs.length; i++ ) {
+      const tab = tabs[i];
+      new Tabs( tab ).init( );
```

Comment:
Run `gulp lint --fix` if you haven't. I think it may strip out the empty space in the method call.

Reply:
Done in https://github.com/cfpb/design-system/pull/502/commits/11e47e89371308e03d646e9c95bb4d50f2b8db6e